auslib/admin/views/releases.py

```diff
@@ -233,9 +233,8 @@ def _put(self, release, changed_by, transaction):
         if dbo.releases.getReleases(name=release, limit=1):
             data_version = form.data_version.data
             try:
-                dbo.releases.updateRelease(name=release, blob=blob,
-                                           product=form.product.data, changed_by=changed_by,
-                                           old_data_version=data_version, transaction=transaction)
+                dbo.releases.update({"name": release}, {"data": blob, "product": form.product.data}, changed_by=changed_by,
```

Comment:
Nit, In some of these insert/update/delete calls we have where=blah style arguments, which helps readability.